### PR TITLE
Delete Polls

### PR DESCRIPTION
**Pull request by Shiyuan-Huang-23**
## Overview
Implemented poll deletion for admins.

  ## Changes Made
- Admin polls now have the more options dots in the header that, when clicked, will bring up the delete poll popup
- The background is dimmed when the popup appears, and clicking outside of the popup will close the it
- Deleting all the polls on a given day will bring you back to `PollsDateActivity`
- Going back to `PollsDateActivity` will refresh polls 

 ## Test Coverage
Tested on Pixel 2 @ API 27.

  ## Related PRs or Issues 
- (#58)

 ## Screenshots 
Screen recording in #pollo-android.

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt [hunk 1]
```diff
@@ -132,21 +135,26 @@ class PollsActivity : AppCompatActivity(), SocketDelegate {
...
+        }
+
         runOnUiThread {
             adapter.notifyItemRemoved(removePollID)
             adapter.notifyDataSetChanged()
             if (removePollID == polls.size) {
                 linearLayoutManager.scrollToPosition(polls.size - 1)
                 currentPollView.text = "${polls.size} / ${polls.size}"
-            }else{
+            } else{
```
Comment (kevinsun-dev) [inline app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt:155]: Yes, I'mma do a Young. Consistent linting please! If there's a space before the parentheses on other lines, do the same here. Alternatively, just CTRL+ALT+L on ever file before committing to run the auto-linter.

File: app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt [hunk 2]
```diff
@@ -103,7 +106,7 @@ class PollsActivity : AppCompatActivity(), SocketDelegate {
                 firstCalendar.get(Calendar.YEAR) == secondCalendar.get(Calendar.YEAR)
 
         if (!datesSameDay) return // No need to handle a new poll if it is not the same day
-        if (poll.id == polls[polls.size - 1].id) return // No need to handle a new poll if it already exists
+        if (polls.isNotEmpty() && poll.id == polls[polls.size - 1].id) return // No need to handle a new poll if it already exists
```
Comment (kevinsun-dev) [inline app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt:109]: Perhaps put the comments on the line before. They end up sticking out a bit far.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -72,6 +72,10 @@
     <string name="save_as_draft">Save as Draft</string>
     <string name="start_poll">Start Question</string>
 
+    <!-- Poll Options -->
+    <string name="close_poll_options_content_description">Close Poll Options</string>
```
[the remark this answers is not in the document]
Comment (jmf373) [reply app/src/main/res/values/strings.xml:76]: Nit: use or delete this (👏 for using content descriptions, yay accessibility)

File: app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt (hunk 1 above)
Comment (Shiyuan-Huang-23) [reply app/src/main/java/com/cornellappdev/android/pollo/Polls/PollsActivity.kt:155]: Makes sense. Will do!

**Pull request merged**